pkgs/top-level/all-packages.nix

```diff
@@ -28380,6 +28380,8 @@ with pkgs;
 
   renoise = callPackage ../applications/audio/renoise {};
 
+  redux = callPackage ../applications/audio/redux {};
```

Comment:
The standard is to use `{ }` in `all-packages.nix`, although not all packages do so.